installer/resources/pacbot_app/files/DB.sql

```diff
@@ -3277,4 +3277,5 @@ UPDATE cf_Target SET displayName='SQL Server' WHERE targetName in ('cloudsql_sql
 UPDATE cf_Target SET displayName='IAM User' WHERE targetName in ('iamuser','iamusers');
 UPDATE cf_Target SET displayName='KMS Key' WHERE targetName in ('kms','kmskey');
 UPDATE cf_Target SET displayName='Subnet' WHERE targetName in ('subnet','subnets');
-UPDATE cf_Target SET displayName='VM' WHERE targetName in ('virtualmachine','vminstance');
\ No newline at end of file
+UPDATE cf_Target SET displayName='VM' WHERE targetName in ('virtualmachine','vminstance');
+UPDATE cf_Target SET displayName = 'Environment', endpoint = concat(@eshost,':',@esport,'/checkmarx_environment') WHERE dataSourceName in ('checkmarx') AND targetName = 'environment';
```

Comment:
it's ok here, but I would recommend to avoid using `in` when you have specifically only one value to compare against. It's just faster. But here is ok - executing only once per deploy